src/e2e-tests/page-objects/smartthings/bathroomTab.page.js

```diff
@@ -0,0 +1,22 @@
+// bathroomTab.page.js
+import Page from '../page';
+import MenuComponent from '../menu.component';
+
+class BathroomTabPage extends Page {
```

Comment:
shall not extend page because it is not a page, it is more a component, set of options

Reply:
Updated.